config/initializers/job_configurations.rb

```diff
@@ -127,6 +127,11 @@
         cron: cron_1h,
         args: -> { [Time.zone.now - 1.hour] },
       },
+      # Weekly IRS report returning system demand
+      irs_weekly_summary_report: {
+        class: 'IrsWeeklySummaryReport',
+        cron: cron_24h,
+        args: -> { [Time.zone.now] },
       # Backfill service_provider_identities last_consented_at
```

Comment:
this is missing a curly brace
```suggestion
      # Weekly IRS report returning system demand
      irs_weekly_summary_report: {
        class: 'IrsWeeklySummaryReport',
        cron: cron_24h,
        args: -> { [Time.zone.now] },
      },
      # Backfill service_provider_identities last_consented_at
```